model/note/text.go

```diff
@@ -0,0 +1,104 @@
+package note
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/cozy/prosemirror-go/markdown"
+	"github.com/cozy/prosemirror-go/model"
+)
+
+func textSerializer() *markdown.Serializer {
+	nodes := map[string]markdown.NodeSerializerFunc{
+		"paragraph": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderInline(node)
+			state.CloseBlock(node)
+		},
+		"text": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.Text(*node.Text)
+		},
+		"bulletList": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderList(node, "", func(_ int) string { return "- " })
+		},
+		"orderedList": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderList(node, "", func(i int) string { return fmt.Sprintf("%d.", i+1) })
+		},
+		"listItem": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderContent(node)
+		},
+		"heading": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderInline(node)
+			state.CloseBlock(node)
+		},
+		"blockquote": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderContent(node)
+		},
+		"rule": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.Write("---")
+			state.CloseBlock(node)
+		},
+		"hardBreak": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.Write("\n")
+		},
+		"image": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			// Nothing
```

Comment:
Do we have a notion of alt text for images in Notes? We could render something like `[My amazing cat]`

Reply:
No, the prosemirror plugin from atlaskit doesn't have that

Reply:
OK, too bad.